content/doc/book/pipeline/multibranch.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:imagesdir: /doc/book/resources/pipeline
+:hide-uri-scheme:
+:toc:
+
+= Multibranch Pipelines
+
+In the <<jenkinsfile#, previous section>> a `Jenkinsfile` which could be
+checked into source control was implemented. This section covers the concept of
+*Multibranch* Pipelines which build on the `Jenkinsfile` foundation to provide
```

Comment:
@scribblingfox - rather than making comments, would you be will to commit your changes and do a PR?  If that's too much work, no big deal, but it seems like your suggestions so far have basically been spot-on - you might was well just make them yourself.  😄  Either way, thanks for all the great work!